src/ci/scripts/install-wix.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# We use the WIX toolset to create combined installers for Windows, and these
+# binaries are downloaded from https://github.com/wixtoolset/wix3 originally
+
+set -euo pipefail
+IFS=$'\n\t'
+
+source "$(cd "$(dirname "$0")" && pwd)/../shared.sh"
+
+if isWindows; then
+    curl -O https://rust-lang-ci-mirrors.s3-us-west-1.amazonaws.com/rustc/wix311-binaries.zip
```

Comment:
```suggestion
    curl -O "${MIRRORS_BASE}/wix311-binaries.zip"
```
something similar can probably be done throughout many files?

Reply:
Yeah, good catch.